src/libraries/System.DirectoryServices.AccountManagement/src/System/DirectoryServices/AccountManagement/AD/ADStoreCtx_Query.cs

```diff
@@ -931,14 +931,11 @@ public static string DateTimeFilterBuilder(string attributeName, DateTime search
             bool defaultNeeded = false;
 
             ldapSearchValue = ADUtils.DateTimeToADString(searchValue);
-            if (defaultValue != null)
-            {
-                ldapDefaultValue = ADUtils.DateTimeToADString(defaultValue);
-            }
+            ldapDefaultValue = ADUtils.DateTimeToADString(defaultValue);
```

Comment:
@joperezr, would you mind double-checking my changes under directory services?  These all stem from dead code, e.g. fields that always had a given value.

Reply:
I'm not very familiar with S.DS.AM as that is a complete different implementation compared to S.DS.P (AM is implemented on top of ADSI, Protocols on top of wldap32) so I'm happy to take a look but it would be good if @tquerec could take a look at that part too.

Reply:
Changes in both S.DS.P and S.DS.AM looks good to me.